css/homepage.css

```diff
@@ -0,0 +1,99 @@
+body {
+  font-family: "Courier New", Courier, monospace;
+  margin:0px auto;
+  padding:0px auto;
+  background: #333; /* Grey for darkening */
+}
+h1 {
+  border-bottom: solid;
+  border-bottom-color: var(--highlight);
+    font-weight: lighter;
+}
+h2 {
+  position:relative;
```

Comment:
Slightly maddeningly inconsistent in this file, feel like there should be either of the following, rather than both
```
key:value
key: value
```

Reply:
I agree... This is my laziness of autocomplete vs me typing... xD

Reply:
Fixed in https://github.com/Zenloth/zenloth.github.io/commit/6c9e755c336f475bf2c9e708db4e69588c6767f3